lib/command/src/Obelisk/Command/Project.hs

```diff
@@ -312,9 +313,22 @@ nixShellRunConfig root isPure command = do
       , [cs]
       ])
 
+-- | Escape using ANSI C-style quotes $''
+-- This does not work with all shells! Ideally, we would control exactly which shell is used,
+-- down to its sourced configuration, throughout the obelisk environment. At this time, this
+-- is not feasible.
 bashEscape :: String -> String
 bashEscape = BSU.toString . bytes . bash . BSU.fromString
 
+-- | Escape using bourne style shell escaping
```

Comment:
Bourne is a proper noun

Reply:
Also: period to end the sentence

Reply:
you get no punctuation but Bourne will get his name